assets/scss/templates/_reportes.scss

```diff
@@ -129,6 +129,7 @@
 .container-reports {
   width: 100%;
   padding: $dp-spaces--lv10 $dp-spaces--lv5;
+  flex: 1;
```

Comment:
don't work with the `dp-app-container` element only? why need this?

Reply:
Necesitaba que el padre de Todo sea un contenedor `Flex` para que sus contenedores hijos sean elementos `Flex`. 
`Flex` es un shorthand que agrupa 3 propiedades: "flex-grow: factor de crecimiento", "flex-shrink: factor de encogimiento" y "Flex-basis: define el tamaño inicial en su eje". Por defecto todo empieza en 0. Cuando en el contenedor central le coloco flex=1 le estoy diciendo que crezca todo el tamaño disponible.
Esto hace que crezca en proporción y ocupe todo el espacio disponible empujando así al `footer` en su lugar.

Reply:
Buenisimo, gracias!!